### PR TITLE
Feature/us8919 start a group mid pages

### PR DESCRIPTION
**Pull request by JoeKer1**

### CODE REVIEW

File: src/app/components/create-group/create-group-data.service.ts [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
 import { LookupService } from '../../services/lookup.service';
-import { groupCategoryAttributeTypeId } from '../../shared/constants';
+import { SessionService } from '../../services/session.service';
+import { attributeTypes } from '../../shared/constants';
 
 @Injectable()
 export class CreateGroupService {
-    public meetingTimeType: string = 'specific';
     private pageOneInitialized: boolean = false;
+    public meetingTimeType: string = 'specific';
```
Comment (vchtelmakh) [inline src/app/components/create-group/create-group-data.service.ts]: Should this be a constant? Not sure if multi-use.

Comment (JoeKer1) [inline src/app/components/create-group/create-group-data.service.ts]: I don't think so, but you just reminded me of something I forgot to do. basically a group can be flexible or at a specific time. There's no flag on the group to store that it's just the absence of meetingDay, meetingTime, etc

Comment (vchtelmakh) [inline src/app/components/create-group/create-group-data.service.ts]: Sounds good

File: src/app/components/create-group/page-3/create-group-page-3.component.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+</div>
+<div>
+    <pre>{{locationForm.controls['kidsWelcome'].valid}}</pre>
+    <pre>{{createGroupService.meetingIsInPerson}}</pre>
+    <pre>{{createGroupService.group | json }}</pre>
```
Comment (vchtelmakh) [inline src/app/components/create-group/page-3/create-group-page-3.component.html]: testing code

File: src/app/components/create-group/page-3/create-group-page-3.component.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+@Component({
+    selector: 'create-group-page-3',
+    templateUrl: './create-group-page-3.component.html',
+})
+export class CreateGroupPage3Component implements OnInit {
+    public locationForm: FormGroup;
+    private usStatesList: string[] = usStatesList;
+    private isSubmitted: boolean = false;
+
+    private meetingFrequencies = meetingFrequencies;
```
Comment (vchtelmakh) [inline src/app/components/create-group/page-3/create-group-page-3.component.ts]: type

File: src/app/components/create-group/page-4/create-group-page-4.component.ts [hunk 4]
```diff
@@ -0,0 +1,142 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validator, Validators } from '@angular/forms';
+import { Location } from '@angular/common';
+import { Router } from '@angular/router';
+import { PAGINATION_CONTROL_VALUE_ACCESSOR } from 'ngx-bootstrap/pagination/pagination.component';
+import { Observable } from 'rxjs';
+
+import { Address } from '../../../models';
+import { Attribute } from '../../../models';
```
Comment (vchtelmakh) [inline src/app/components/create-group/page-4/create-group-page-4.component.ts]: Combine into one import line.

**Pull request merged**